mne/io/artemis123/artemis123.py

```diff
@@ -32,6 +35,13 @@ def read_raw_artemis123(input_fname, preload=False, verbose=None):
         on the hard drive (slower, requires less memory).
     verbose : bool, str, int, or None
         If not None, override default verbose level (see mne.verbose).
+    pos_fname : str or None (default None)
+        If not None, load digitized head points from this file
+    head_loc : bool (default True)
+        If True attempt to perform initial head localization (compute initial
+        evice to head coordinate transform) using HPI coils. If no
+        HPI coils are in info['dig'] hpi coils are assumed to be in canonical
+        order of fiducial points (nas, rpa, lpa).
```

Comment:
`add_head_trans` it is.

> In this case it should set it to None actually. It's possible (even if unlikely) to get identity in real data, None is used by MNE to mean "could not be computed" which is appropriate here.


` dev_head_t` is set to identity in https://github.com/mne-tools/mne-python/blob/master/mne/io/meas_info.py#L1538. If the desired default/unset behavior is `None` I would think the change should happen there.  I also don;t have an issue the head cords and device cords being the same space (identity transform). I imagine having `dev_head_t=None` may break things that I would like to work even when no coils are being used. Such as plotting the sensor locations or doing dipole fits in phantoms etc.


> I thought in your code you had it do all permutations of coil orderings

That only happens if there are independent measures of the hpi locations (ie from the digitization). without that i can only assume a mapping from hpis to fiducials etc.